package.json

```diff
@@ -35,7 +35,8 @@
     "git:sreset": "git submodule foreach 'git reset --hard'",
     "ngcc": "ngcc",
     "ng": "ng",
-    "start": "npm run ngcc && ng serve --configuration=development",
+    "start": "ng serve --configuration=development",
+    "postinstall": "npm run ngcc",
```

Comment:
Double check that it will not break xmake.

Reply:
I think that we will end up with two different repos the way I see things today with our colleagues.
First we have to move to SAP public space and next the repo will be forked to the private one where we will not have submodules anymore.
Let's see.
For the time being, let's not keep it in full sync with SAP one.

Reply:
Ok, but before it's done I think it's better do not break the current setup. 
And doing an automated synchro between a public repo with submodules to a private one that will incorporate the submodules in the source is going to be difficult and very hard to maintain. Maybe not possible: git cannot handle two different history.
We probably have to discuss why we need an internal mirror and if it's really needed. If yes, find the simplest solution to mirror it automatically.

Reply:
Ok, I'll discuss this with you next week :)